src/backgroundQueue/server.spec.ts

```diff
@@ -51,5 +56,47 @@ describe('makeQueueServer', () => {
       expect(response.statusCode).toBe(HTTP_STATUS_CODES.BAD_REQUEST);
       expect(response.json()).toHaveProperty('type', QueueProblemType.UNSUPPORTED_EVENT);
     });
+
+    describe('makeQueueServerPlugin', () => {
+      const setEnvVars = configureMockEnvVars(REQUIRED_ENV_VARS);
```

Comment:
Note that this will create a second mock of the env vars -- each with its own set of `beforeAll()`s and `beforeEach()`s. That may lead to race conditions that may be hard to debug.

We have `getTestServerFixture`. Let's get the mocker from there.